3_visualize/src/visualize_data.R

```diff
@@ -1,9 +1,9 @@
 ############## Visualize data  ######################
 
 
-plot_results <- function(mendota_file, project_output_dir) {
-              
-                    png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
+plot_results <- function(mendota_proc_data, project_output_dir) {
+ 
+                    png(mendota_proc_data, project_output_dir = file.path(project_output_dir, names = 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
this looks wrong (and not like what i remember from our most recent phone call) - check again for updates to commit?